instance/tests/managament/test_activity_csv.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2016 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance - Activity CSV unit tests
+"""
+# Imports #####################################################################
+
+from django.core.management import call_command
+from django.utils.six import StringIO
+from django.test import TestCase
+
+
+# Tests #######################################################################
+
+class ActivityCSVTestCase(TestCase):
+    """
+    Test cases for OpenStack helper functions
```

Comment:
Hah, I actually copied the entire header and class structure straight from another test as a starting point and forgot to change this. I used your suggestion to fix this, thanks. :)